### PR TITLE
Make the label follow the seedType

### PR DESCRIPTION
**Pull request by tcbyrd**
@cheshire137 Super minor fix here! Such amazing work on this! ✨ ✨ 
typefix

### CODE REVIEW

File: src/components/seed-search-form.jsx [hunk 1]
```diff
@@ -197,7 +197,7 @@ class SeedSearchForm extends React.Component {
         <h3 className="title is-3">Step 1: Seed your playlist</h3>
         <div className="control">
           <label className="label" htmlFor="seed">
-            Find songs like:
+            Find {seedType}s like:
```
Comment (cheshire137) [inline src/components/seed-search-form.jsx]: Hmm, I think this isn't right, actually: whether you choose an artist seed or a track seed, the ["get recommendations" endpoint](https://developer.spotify.com/web-api/get-recommendations/) will return tracks. So "Find songs" is always accurate. 🤔

**Comment by cheshire137** [comment]
The phrasing might not be clear here, though. You're searching either songs or artists, but the goal is to find songs like that song/artist. Any thoughts on making that clearer? Maybe your change makes sense because in that first form, you are looking for either songs or artists. It's just the end goal that is always songs.

**Comment by tcbyrd** [comment]
AHHH. I see what you're saying. I think this is why it confused me:
image

So you do ultimately end up finding songs similar to that artist, but you aren't selecting a song to pick from. 

But now that I look at it more, the sliders in the next step appear to just be defaulted (i think) since the artist object doesn't have audio features.
image

And you can already search by an artist name in the song search:
image

So since the artist object doesn't have any audio features, maybe it's not needed?

**Comment by cheshire137** [comment]
> since the artist object doesn't have any audio features, maybe it's not needed?

I was trying to offer similar support for what https://developer.spotify.com/web-api/get-recommendations/ allows:

> Create a playlist-style listening experience based on seed artists, tracks and genres.

> Recommendations are generated based on the available information for a given seed entity and matched against similar artists and tracks. If there is sufficient information about the provided seeds, a list of tracks will be returned together with pool size details.

How about keeping your change but also adding an introductory paragraph? So the label updates to indicate whether you're searching tracks or artists, and we have a paragraph above that with something like "Create a playlist with songs similar to a particular song or artist."

You could change your code slightly to not show just the seedType, but rather if the seedType is 'track' show 'songs'. I've been trying to call them 'songs' throughout the app; I think switching between songs/tracks could be confusing.

**Comment by tcbyrd** [comment]
How about using some of Spotify's wording ("Get Recommendations based on...") right in the label?
based-on

Also I'm cooking up an idea for genre search. I realized it already works out of the box by typing "genre:jazz" in the search box and you get results of either artists or songs in that genre! I'll open a separate PR for that since I'd like to get your ideas on how it would work.

**Comment by cheshire137** [comment]
> How about using some of Spotify's wording

Oh that's better! Have just the label, so less text for users to [not] read. 😆 

> Also I'm cooking up an idea for genre search.

Sure! That 'get recommendations' endpoint works with genres, it says in the API docs, I just hadn't figured out a good way to expose it/explain it in the UI since I don't think there's a genre search where I can type "ra" and get "rap" autocompleted, like you can for artists and tracks. If you've got UI ideas, I'm interested!

**Review by cheshire137 — APPROVED** [review]

**Pull request merged**